examples/high_frequency/emc/busbar.py

```diff
@@ -117,18 +117,66 @@
 print(q3d.net_sources("Bar2"))
 print(q3d.net_sources("Bar3"))
 
+
+# ## Create Matrix Reduction Operations
+#
+# Series of Bar1 and Bar2
+
+mr_series = q3d.insert_reduced_matrix(
+    operation_name="JoinSeries",
+    assignment=["Sink1", "Source2"],
+    reduced_matrix="MR_1_Series",
+    new_net_name="Series1",
+)
+
+# Add Parallel with Bar3
+
+mr_series.add_operation(
+    operation_type="JoinParallel",
+    source_names=["Source1", "Source3"],
+    new_net_name="SeriesPar",
+    new_source_name="src_par",
+    new_sink_name="snk_par",
+)
+
+# Series of Bar1 and Bar2
+
+mr_series2 = q3d.insert_reduced_matrix(
+    operation_name="JoinSeries",
+    assignment=["Sink1", "Source2"],
+    reduced_matrix="MR_2_Series",
+    new_net_name="Series2",
+)
+
+# Add Series with Bar3
+
+mr_series2.add_operation(
+    operation_type="JoinSeries",
+    source_names=["Sink3", "Source1"],
+    new_net_name="MR_2_Series1",
+)
+
 # Create the solution setup and define the frequency range for the solution.
 
-setup1 = q3d.create_setup(props={"AdaptiveFreq": "100MHz"})
-sweep = setup1.add_sweep()
-sweep.props["RangeStart"] = "1MHz"
-sweep.props["RangeEnd"] = "100MHz"
+freq_sweep_name = "my_sweep"
+setup1 = q3d.create_setup(props={"AdaptiveFreq": "1000MHz"})
+sweep = setup1.add_sweep(name=freq_sweep_name)
+sweep.props["RangeStart"] = "0Hz"
+sweep.props["RangeEnd"] = "20GHz"
 sweep.props["RangeStep"] = "5MHz"
 sweep.update()
+sweep.update()
+
+# ## Analyze
+#
+# Solve the setup.
+
+q3d.analyze(cores=NUM_CORES)
+q3d.save_project()
 
 # ### Set up for postprocessing
 #
-# Specify the traces to display after solving the model.
+# Specify the traces to display after solving the model. Capacitances - Original Matrix
 
 data_plot_self = q3d.matrices[0].get_sources_for_plot(
```

Comment:
```suggestion
original_matrix_self = q3d.matrices[0].get_sources_for_plot(
```
and the other one below that I can't comment: data_plot_mutual -> original_matrix_mutual